_data/sites.json

```diff
@@ -7027,17 +7027,16 @@
 
     {
         "name": "GNOME",
-        "url": "https://www.gnome.org/privacy-policy/",
-        "difficulty": "hard",
-        "email": "info@gnome.org",
-        "notes": "Account deletion requires contacting Customer Support.",
-        "notes_tr": "Hesabınızın silinmesi için müşteri hizmetleri ile iletişime geçmeniz gerekir.",
-        "notes_fr": "Pour supprimer votre compte contactez le support.",
-        "notes_it": "La cancellazione account richiede di contattare il servizio clienti.",
-        "notes_de": "Account-Löschung erfordert es den Kundensupport zu kontaktieren",
-        "notes_pt_br": "Para deletar a sua conta contate a assistência ao cliente",
-        "notes_cat": "Per esborrar el seu compte contacti amb l'assistència al client",
-        "notes_es": "Para borrar tu cuenta contacta con asistencia al cliente",
+        "url": "https://extensions.gnome.org/accounts/settings",
+        "difficulty": "medium",
+        "notes": "In the section Account removal, you need to accept the option delete your account, and input your password.",
```

Comment:
Why not easy? Seems very straightforward

Reply:
I thought inputting the password was considered to be the "additional step". Because [the `CONTIBUTING.md` stated that](https://github.com/jdm-contrib/jdm/blob/master/CONTRIBUTING.md?plain=1#L10)

> medium: Sites that do allow account deletion but require you to perform additional steps

Reply:
Ah, it's a grey area, but I think inputting the password is easy enough. I think a more convoluted extra step would be to get an e-mail then using that e-mail to confirm and etc.